tcpdf.php

```diff
@@ -6495,14 +6495,16 @@ public function Write($h, $txt, $link='', $fill=false, $align='', $ln=false, $st
 				$shy = false;
 				// account for margin changes
 				if ((($this->y + $this->lasth) > $this->PageBreakTrigger) AND ($this->inPageBody())) {
-					$this->AcceptPageBreak();
-					if ($this->rtl) {
-						$this->x -= $margin['R'];
-					} else {
-						$this->x += $margin['L'];
+					if ($this->AcceptPageBreak())
+					{
+						if ($this->rtl) {
```

Comment:
should this be reached if AcceptPageBreak is false ?

Reply:
I don't think this code should be run if `AcceptPageBreak` is false.  My understanding is that `AcceptPageBreak` will return false if a page break is not added.  If a page break is not added, there should be no need to adjust the margin.